apps/web/src/features/earn/components/VaultRedeemConfirmation/index.tsx

```diff
@@ -8,6 +8,7 @@ import { DataTable } from '@/components/common/Table/DataTable'
 import { DataRow } from '@/components/common/Table/DataRow'
 import IframeIcon from '@/components/common/IframeIcon'
 
+// TODO: Check if additional rewards can actually appear for a withdraw/redeem
```

Comment:
When are we going to check this?

Reply:
From what I can see is that we do fetch extra rewards on the CGW for every vault interaction but for redeems/withdraws the claimable amount is always 0 thats why we don't display it. I remember Kiln mentioning that claiming these extra rewards will remain as a separate step but we should check with them.